packages/one-app-runner/src/startApp.js

```diff
@@ -0,0 +1,89 @@
+      ...process.env.NO_PROXY && { NO_PROXY: process.env.NO_PROXY },
+    };
+    return Object.keys(environmentVariablesWithProxyAdditions).reduce((accumulator, currentValue) => `${accumulator} -e ${currentValue}=${environmentVariablesWithProxyAdditions[currentValue]}`, '');
+  };
+
+  const generateModuleMountsArgs = (modules) => {
+    let args = '';
+    if (modules && modules.length > 0) {
+      args = modules.reduce((accumulator, currentValue) => {
+        const moduleRootDir = path.basename(currentValue);
+        return `${accumulator} -v ${currentValue}:/opt/module-workspace/${moduleRootDir}`;
+      }, '');
+    }
+
+    return args;
+  };
+
+  const generateSetMiddlewareCommand = (pathToMiddlewareFile) => {
+    if (pathToMiddlewareFile) {
+      const pathArray = pathToMiddlewareFile.split(path.sep);
+      return `npm run set-middleware /opt/module-workspace/${pathArray[pathArray.length - 2]}/${pathArray[pathArray.length - 1]} &&`;
+    }
+    return '';
+  };
+
+  const generateSetDevEndpointsCommand = (pathToDevEndpointsFile) => {
+    if (pathToDevEndpointsFile) {
+      const pathArray = pathToDevEndpointsFile.split(path.sep);
+      return `npm run set-dev-endpoints /opt/module-workspace/${pathArray[pathArray.length - 2]}/${pathArray[pathArray.length - 1]} &&`;
+    }
+    return '';
+  };
+
+  const generateUseMocksFlag = (shouldUseMocks) => (shouldUseMocks ? '-m' : '');
+
+  const generateServeModuleCommands = (modules) => {
+    let command = '';
+    if (modules && modules.length > 0) {
+      modules.forEach((modulePath) => {
+        const moduleRootDir = path.basename(modulePath);
+        command += `npm run serve-module /opt/module-workspace/${moduleRootDir} &&`;
+      });
+    }
+    return command;
+  };
+
+  const command = `docker pull ${appDockerImage} && docker run -t -p 3000-3005:3000-3005 -e NODE_ENV=development ${generateEnvironmentVariableArgs(envVars)} ${generateModuleMountsArgs(modulesToServe)} ${appDockerImage} /bin/sh -c "${generateServeModuleCommands(modulesToServe)} ${generateSetMiddlewareCommand(parrotMiddlewareFile)} ${generateSetDevEndpointsCommand(devEndpointsFile)} node lib/server/index.js --root-module-name=${rootModuleName} --module-map-url=${moduleMapUrl} ${generateUseMocksFlag(parrotMiddlewareFile)}"`;
+  const dockerProcess = spawn(command, { shell: true });
+  dockerProcess.on('error', () => {
+    throw new Error(
+      'Error running docker. Are you sure you have it installed? For installation and setup details see https://www.docker.com/products/docker-desktop'
+    );
+  });
+  [
+    'SIGINT',
+    'SIGTERM',
+  ].forEach((signal) => {
+    // process is a global referring to current running process https://nodejs.org/api/globals.html#globals_process
+    process.on(signal, () => 'noop - just need to pass signal to one app process so it can handle it');
+  });
+
+  if (outputFile) {
+    const logFileStream = fs.createWriteStream(outputFile);
```

Comment:
did you figure out how to force colour ? if not can you add an issue

Reply:
I did figure it out 🎉